### PR TITLE
Issuers as extension

### LINKED ISSUE #1957: Move Isuers as extension
**Description**

We replace our current implementation with the extensibility option.

**Acceptance Criteria**

- [ ] Validate with Product Owner if the current implementation is correct
- [ ] Replace the current implementation with extensibility
- [ ] Deploy to stage and Test all the edge cases on stage
- [ ] Deploy to production
- [ ] Remove old codebase
- [ ] Adjust Tests to use extensions

### PR DESCRIPTION
**Pull request by chriskari**
**Description**

Changes proposed in this pull request:

- added issuers as extension
- adjusted tests
- removed old codebase

**Related issue(s)**
closes #1957

### CODE REVIEW

File: src/components/Clusters/views/EditCluster/AuthenticationDropdown.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function AuthenticationTypeDropdown({ type, setType }) {
       options={options}
       selectedKey={type}
       onSelect={(_, selected) => setType(selected.key)}
-      placeholder={t('issuers.placeholders.type')}
+      placeholder={t('clusters.placeholders.type')}
```
Comment (mrCherry97) [inline src/components/Clusters/views/EditCluster/AuthenticationDropdown.js:17]: It's unnecessary, can be removed

File: public/i18n/en.yaml [hunk 2]
```diff
@@ -138,6 +138,8 @@ clusters:
   description-visibility: Description is shown on the Clusters Overview list
   disconnect: Cluster disconnected
   edit-cluster: Edit Cluster
+  placeholders: 
+    type: Select Issuer type
```
Comment (mrCherry97) [inline public/i18n/en.yaml:142]: This too can be removed

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: src/components/Clusters/views/EditCluster/AuthenticationDropdown.js (hunk 1 above)
Comment (chriskari) [reply src/components/Clusters/views/EditCluster/AuthenticationDropdown.js:17]: sorry what exactly do you mean, just line 17 ?

File: src/resources/index.js [hunk 3]
```diff
@@ -15,7 +15,6 @@ import * as Pods from './Pods';
 import * as PersistentVolumeClaims from './PersistentVolumeClaims';
 import * as OAuth2Clients from './OAuth2Clients';
 import * as NetworkPolicies from './NetworkPolicies';
-import * as Issuers from './Issuers';
```
Comment (mrCherry97) [inline src/resources/index.js:18]: line 64 in this file needs to be removed

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
I forgot, we need to wait until https://github.com/kyma-project/busola/pull/2277 will be merged

**Review by grego952 — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]
